regression-test/suites/datatype_p0/decimalv3/test_load.groovy

```diff
@@ -104,4 +104,59 @@ suite("test_load") {
         exception "error"
     }
     qt_decimalv3_insert "select * from test_decimalv3_insert order by 1, 2;"
+
+    // fix
+    sql """ set enable_nereids_dml=true; """
+    sql """
+        drop TABLE if exists test_sys_update_basic_test_update_decimal_tb;
+    """
+    sql """
+        CREATE TABLE test_sys_update_basic_test_update_decimal_tb (
+          k1 DECIMAL(10, 5) NULL, 
+          v1 DECIMAL(10, 5) NULL
+        ) UNIQUE KEY(k1) DISTRIBUTED BY HASH(k1) BUCKETS 5 PROPERTIES (
+          "replication_allocation" = "tag.location.default: 1"
+        );
+    """
+    sql """
+        insert into test_sys_update_basic_test_update_decimal_tb values
+            (1.001, 2.002), (1.002, 0.00000002), (1.003, 0.100000001), (1.004, 0.100044001), (1.005, 0.100045001);
```

Comment:
Insert:
0.00000002 ===> 0.00000

Update:
"0.00000001" ===> 0.00000

still the same result, we still do not know `update set` stmt work correct or not.

maybe insert 1.00000001, then update 2.00000002 ?